composer.json

```
@@ -18,7 +18,7 @@
     },
     "require-dev": {
         "barryvdh/laravel-debugbar": "^3.9",
-        "fzaninotto/faker": "^1.0",
+        "fakerphp/faker": "^1.0",
```

Let's align this with the `laravel/laravel` version
```suggestion
        "fakerphp/faker": "^1.23",
```